### PR TITLE
Bug 1460990 - show underlying username in several places where user may look

### PR DESCRIPTION
**Pull request by jwforres**
Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1460990

### CODE REVIEW

**Comment by jwforres** [comment]
about-user
cli-and-logout

**Comment by jwforres** [comment]
open to other suggestions

**Comment by spadgett** [comment]
Does the logout menu grow wider if the ID is something long like an email address?

Screenshots look good.

**Comment by jwforres** [comment]
Yeah the menu seems to fit to the content

**Comment by spadgett** [comment]
Suggest we only show in the menu if it's different than what we show in the banner already.

**Comment by jwforres** [comment]
updated to only show it in logout if fullName exists and is different

File: app/scripts/extensions/nav/userDropdown.js [hunk 1]
```diff
@@ -1,12 +1,16 @@
...
     extensionRegistry
       .add('nav-user-dropdown', function() {
+        var msg = 'Log out';
+        if ($rootScope.user.fullName && $rootScope.user.fullName !== $rootScope.user.metadata.name) {
+          msg += ' (' + $rootScope.user.metadata.name + ')';
+        }
         return [{
           type: 'dom',
-          node: '<li><a href="logout">Log out</a></li>'
+          node: '<li><a href="logout">' + msg + '</a></li>'
```
Comment (spadgett) [inline app/scripts/extensions/nav/userDropdown.js]: _.escape(msg)

Comment (jwforres) [inline app/scripts/extensions/nav/userDropdown.js]: fixed

**Comment by spadgett** [comment]
[merge][severity: blocker]

**Pull request merged**